helpers/components/addOrderedIds.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const _ = require('lodash');
+
+/**
+ * Add ordered ids to components within a componentlist
+ * @param {Array} content   list of components
```

Comment:
Let's use the notation `Object[]` to indicate we're expecting an array of objects